packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/MultiValueField.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import MultiValueField, { MultiValueFieldProps } from './MultiValueField'
+import { ColumnTypeEnum } from '@sage-bionetworks/synapse-types'
+
+const meta: Meta<MultiValueFieldProps> = {
+  title: 'Components/MultiValueField',
+  component: MultiValueField,
+  decorators: [
+    function UseStateWrapper(Story, args) {
+      const [value, setValue] = useState<typeof args.value>(args.value)
+      return <Story {...args} value={value} onChange={setValue} />
+    },
+  ],
+}
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const StringList: Story = {
+  args: {
+    value: ['foo', 'bar', 'baz'],
+    columnType: ColumnTypeEnum.STRING_LIST,
+  },
+}
+export const IntegerList: Story = {
+  args: {
+    value: [-5, 20, 0],
+    columnType: ColumnTypeEnum.INTEGER_LIST,
+  },
+}
+export const DateList: Story = {
+  args: {
+    value: ['2023-10-05T14:48:00.000Z', '2024-03-11T07:24:39.000Z'],
```

Comment:
Based on the processing in DefaultValueListField, I was expecting these to be Unix timestamps with millisecond precision. Does MultiValueField handle Date lists with other datetime formats as well?

Reply:
`MultiValueField` consumes ISO strings (internally it uses React JSON Schema Form, and JSON Schema date-time uses ISO strings). `DefaultValueListField` converts integers to ISO strings to pass into `MultiValueField`, and then transforms ISO strings back to integers.

Since this story just touches MultiValueField, ISO timestamps are appropriate. Managing all of these different levels of translation can get confusing, and it makes it pretty easy to write bugs 🙃

Reply:
Oh that is confusing! So that's also why the 'x' argument is not needed [here](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/803#discussion_r1520132015), since `MultiValueField` returns an ISO string `onChange`, which can be directly converted to a Day.js object with the default `dayjs` parse? If that's right, would it be worth adding a comment with that context?

Reply:
Oh just saw your comment on that point -- thanks for helping me understand, Nick!

Reply:
Just did!